### DISCUSSION TITLE
[Icon Suggestion] `chevron-down-up`

### DISCUSSION THREAD

**Discussion by gukj** [tailwindlabs/heroicons#942] [Ideas]
There's an icon for two chevrons pointing away from each other (`chevron-up-down`), I would like to see an icon with the chevrons pointing towards each other as well — the `chevron-down-up` 🎉

I missed it when building an interface with an expand/collapse toggle. 

<img width="1308" alt="Screenshot 2023-02-24 at 15 51 27" src="https://user-images.githubusercontent.com/8404408/221209387-fff4f07b-5cb8-476d-9507-cd6f1114bdfe.png">

**Comment by lawik** [comment]
I would love the same for left and rights as well. Expand/contract.

**Reply by adam-binks** [reply]
You could rotate the existing icons for this :)

**Comment by ilovesoup20** [comment]
```
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M8.25 18.75 L12 15 L15.75 18.75 M8.25 5.25 L12 9 L15.75  5.25" stroke="#0F172A" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

Here's the svg for `chevron-down-up`

**Comment by ilovesoup20** [comment]
Created a pull request for adding chevron-down-up icon
https://github.com/tailwindlabs/heroicons/pull/1239#issue-2616235358

**Comment by ndri** [comment]
I added `chevron-down-up` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=chevron-down-up

**Reply by ilovesoup20** [reply]
Awesome, thanks!
